nss-cohort-40/bangazon-ecommerce-client-upperclasstwits
# Product Form

# Description
Built out product form to add a new product. This does not include restrictions to price nor logic to adjust quantity with purchase nor logic to update quantity. Those were separate tickets. This simply allows new product to be added.
Fixes #1 (Ticket #1)
## Type of change
Built out basic add product form. Included a few adjustments to the product view on django, already merged to that master. 
Please delete options that are not relevant.
# Testing Instructions
Go to localhost:3000/products/new and add a product. Check your database that it is now there with the correct info. Error will occur after hitting submit button - item should be in your db but the route re-directed (localhost:300/products which is the list view) to after that hasn't been merged yet.
# Checklist:
